### ISSUE TITLE
Language-specific links should default to Universal definitions if page doesn't exist

### ISSUE THREAD

**Issue by py3ams** [UniversalDependencies/docs#149]
After removing the Animacy page from the Swedish features, the link in the feature table was directed at the French definition. I have also seen links in the Swedish dependency table directed at English definitions when the Swedish page doesn't exist.

**Comment by dan-zeman** [comment]
I believe that the correct way to go here is to edit the HTML table. Instead of `<a>Animacy</a>`, there ought to be `<strike><tt>Animacy</tt></strike>`. This is what I did with the `Definite` feature in Czech.

**Comment by spyysalo** [comment]
I agree with @dan-zeman's solution for this specific case, but the auto-linking to a page for a different language is a misfeature at best. I think the corresponding page on the "universal" side would be a better fallback if a language-specific page is not found. Alternatively, this could just skip linking and report an error.

**Comment by jnivre** [comment]
This seems to have been resolved, so I'm closing the issue.
